autogen/versions.tf.tmpl

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2021 Google LLC
+ * Copyright 2022 Google LLC
```

Comment:
nit: we should bump the min TPG/B version here to 4.10 since thats when support was added. 

https://github.com/hashicorp/terraform-provider-google-beta/blob/main/CHANGELOG.md#4100-february-7-2022

Reply:
It was bumped via https://github.com/terraform-google-modules/terraform-google-lb-http/pull/281 so we should be good.